src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -105,6 +92,10 @@ export class SkyFileAttachmentComponent implements ControlValueAccessor, AfterVi
 
   public rejectedOver: boolean = false;
 
+  /**
+   * Indicates if the input must have a file to be valid. This property accepts boolean values.
```

Comment:
I'm a little confused by "to be valid." What is considered valid? The single file attachment? Or the form that it is included on? I mean, is the validation specific to the single file attachment? Or is it specific to a form, to indicate that users must attach a file before they can complete the form? The latter makes more sense to me, but the current wording seems like the validation is specific to the input. Also, in https://github.com/blackbaud/skyux-forms/pull/77, we explain the impact of setting `required` to `true`. Is there any reason not to include that information in the description? Assuming my understanding of all of this is correct, what do you think of fleshing out this description to:

"Indicates whether the input is required for form validation. When you set this property to `true`, the component adds `aria-required` and `required` attributes to the `input` element, and both template-driven forms and reactive forms show an invalid state until the `input` element is complete. This property accepts a `boolean` value."

Reply:
Yeah that's all accurate, although I'm not sure we have to point out that both reactive and template-driven forms are supported. Isn't that just the assumption that we would support them both?

Reply:
Good point. The description in https://github.com/blackbaud/skyux-forms/pull/77 called out both reactive and template-driven forms, and I included it here without fully thinking it through. Since we don't really need to call that out, we can tighten up to:

"Indicates whether the input is required for form validation. When you set this property to true, the component adds `aria-required` and `required` attributes to the `input` element so that forms display an invalid state until the `input` element is complete. This property accepts a boolean value."

Reply:
Done.